dotnet/src/Microsoft.AutoGen/Agents/Services/Grpc/GrpcGateway.cs

```diff
@@ -33,6 +36,7 @@ public GrpcGateway(IClusterClient clusterClient, ILogger<GrpcGateway> logger)
         _clusterClient = clusterClient;
         _reference = clusterClient.CreateObjectReference<IGateway>(this);
         _gatewayRegistry = clusterClient.GetGrain<IRegistryGrain>(0);
+        _subscriptions = clusterClient.GetGrain<ISubscriptionsGrain>(0);
```

Comment:
Are we expecting the singeliton for this class

Reply:
managed by Orleans